### PR TITLE
Implemented wallet context for handling wallet notifications

### PR DESCRIPTION
**Pull request by Malak67**

### CODE REVIEW

File: services/community-site/src/pages/RunNode/Modal/EditValidator.tsx [hunk 1]
```diff
@@ -50,21 +52,8 @@ const EditValidator = ({
...
-        setStep(2);
-        await res.wait();
-        setLoading(false);
-      } catch (e) {
-        // eslint-disable-next-line no-console
-        console.error(e);
-        setLoading(false);
-      }
+      onSuccess();
+      asyncCallback(setValidatorInfo, [validator.address, payload.description, payload.endpoint]);
```
Comment (rattrap) [inline services/community-site/src/pages/RunNode/Modal/EditValidator.tsx:56]: Why not giving it a function as a param? Seems cleaner and simpler

**Review by rattrap — CHANGES_REQUESTED** [review]

File: services/community-site/src/pages/RunNode/Modal/EditValidator.tsx (hunk 1 above)
Comment (Malak67) [reply services/community-site/src/pages/RunNode/Modal/EditValidator.tsx:56]: But I am. setValidatorInfo is the function, and the second param is the array of params the setValidatorInfo requires.

Comment (Malak67) [reply services/community-site/src/pages/RunNode/Modal/EditValidator.tsx:56]: Or maybe I don`t understand what you are suggesting

Comment (VargaElod23) [reply services/community-site/src/pages/RunNode/Modal/EditValidator.tsx:56]: I guess what Leo meant was a single parameter as such: `(args: any[]) => Promise<any>`

Comment (VargaElod23) [reply services/community-site/src/pages/RunNode/Modal/EditValidator.tsx:56]: With that, we could pass the methods we define on hooks directly into it like importing setValdiatorInfo() and then doing `asyncCallback(setValdiatorInfo([description, endpoint]))`

**Review by VargaElod23 — COMMENTED** [review]
Left some clarifications, besides that it feels okay

File: services/community-site/src/pages/RunNode/Modal/EditValidator.tsx (hunk 1 above)
Comment (Malak67) [reply services/community-site/src/pages/RunNode/Modal/EditValidator.tsx]: Got it, changed

**Review by VargaElod23 — APPROVED** [review]

**Pull request merged**